### PR TITLE
adding update function for sunburst model,

### PR DESCRIPTION
**Pull request by Sghazzawi**
fix for https://github.com/novus/nvd3/issues/1145

### CODE REVIEW

File: src/models/sunburst.js [hunk 1]
```diff
@@ -59,6 +59,15 @@ nv.models.sunburst = function() {
 
             var g = wrapEnter.selectAll('nv-sunburst');
 
+            chart.update = function() { 
+              if( duration === 0 )
+                 container.call(chart);
+               else
+                 container.transition().duration(duration).call(chart);
```
Comment (liquidpele) [inline src/models/sunburst.js]: Please make sure the indention is 4 spaces... the "if" and "else" aren't even lined up here...   

Also, please use brackets for if/else in call cases to match the rest of the code style that we're going for (jslint/jshint rules).

**Comment by liquidpele** [comment]
well, lineChart and sunburstChart look good, but it's still not changed in sunburst.js where I originally commented ;)

**Comment by Sghazzawi** [comment]
sorry about that.

**Comment by liquidpele** [comment]
great, thanks for contributing!

**Comment by Crenshinibon** [comment]
It's still not updating, or do I miss something?

**Pull request merged**